Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1

```diff
@@ -6,10 +6,20 @@ Clear-Host
 [Boolean] $sendSlackMessage =  $args[3]
 [string] $relativityVersionReleaseName = ''
 
-[string] $server = 'solutionsnapshotapi.azurewebsites.net'
+[string] $productionEnvironmentName = 'Production'
+[string] $stagingEnvironmentName = 'Staging'
+[string] $developEnvironmentName = 'Develop'
 
-[string] $CreateRelativityVersionAsyncUrl = "https://$($server)/api/external/CreateRelativityVersionAsync"
-[string] $updateApplicationVersionAsyncUrl = "https://$($server)/api/external/UpdateApplicationVersionAsync"
+[string] $server_production = 'solutionsnapshotapi.azurewebsites.net'
```

Comment:
we have custom url's for production. we should use them instead - https://api.solutionsnapshot.relativity.com

That way we are also testing if the custom url's are functioning properly.

Reply:
Done